boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -30,6 +30,14 @@
 		nordic,iset-gpios = <&gpio0 23 GPIO_ACTIVE_HIGH>;
 	};
 
+	board_id: board_id {
+		compatible = "voltage-divider";
+		io-channels = <&adc 0>;
+		output-ohms = <20000>;
+		full-ohms = <47000>;
```

Comment:
Resistor values cannot be here as different board versions have different values. Suggest to remove this.

Reply:
Each board revision must be defines separately, and an overlay provided with the correct values.

Reply:
https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#multiple-board-revisions ?

Reply:
Suggest we do that in another PR. This requires some discussion and would change the way we work with the boards.

Reply:
I adjusted current code in sdk-nrf so that it works with that. resistor values are actually not relevant, as you seem to store a table with adc raw values.

Reply:
The voltage divider values are actually not relevant at all, because you just care about the voltage it generates (from a constant source) to identify the board. However, voltage-divider compatible comes handy here.